src/components/CategoryFilter.js

```diff
@@ -3,7 +3,9 @@ import PropTypes from 'prop-types';
 
 function CategoryFilter(props) {
   const category = useRef();
-  const { filt } = props;
+  const cat2 = useRef();
+  // const { filt } = props;
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))